src/OrangePaymentSlipData.php

```diff
@@ -153,20 +169,35 @@ public function getReferenceNumber()
      * @param string $bankingCustomerId The banking customer ID.
      * @return $this The current instance for a fluent interface.
      * @throws DisabledDataException If the data is disabled.
+     * @throws PaymentSlipException If the length is greater than 10 digits
      */
-    public function setBankingCustomerId($bankingCustomerId)
+    public function setBankingCustomerId($bankingCustomerId, $bankingCustomerIdLength = 6)
     {
         if (!$this->getWithBankingCustomerId()) {
             throw new DisabledDataException('banking customer ID');
         }
-        // TODO check length (exactly 6)
+        
+        if ($bankingCustomerIdLength > 10) {
```

Comment:
We should not use magic numbers, I prefer to have a constant for that, as this number cannot be changed. E.g. ``static::BANKING_CUST_ID_MAX_LENGTH``